src/modules/Security.rb

```diff
@@ -363,6 +340,80 @@ def read_kernel_settings
         val = default_value if val.nil? || val == ""
         @Settings[key] = val
       end
+
+      log.debug "Settings (after #{__callee__}): #{@Settings}"
```

Comment:
No, my point is that I'd prefer something readable even for non-developers (admins) , e.g. `after reading /etc/sysctl.conf file` is probably be more understandable than the internal function name.